tests/bundle/Functional/ImageVariationTest.php

```diff
@@ -0,0 +1,147 @@
+    <ParentLocation href="/api/ezp/v2/content/locations/1/2" />
+    <priority>0</priority>
+    <hidden>false</hidden>
+    <sortField>PATH</sortField>
+    <sortOrder>ASC</sortOrder>
+  </LocationCreate>
+  <Section href="/api/ezp/v2/content/sections/3" />
+  <alwaysAvailable>true</alwaysAvailable>
+  <remoteId>{$string}</remoteId>
+  <User href="/api/ezp/v2/user/users/14" />
+  <modificationDate>2012-09-30T12:30:00</modificationDate>
+  <fields>
+    <field>
+      <fieldDefinitionIdentifier>name</fieldDefinitionIdentifier>
+      <languageCode>eng-GB</languageCode>
+      <fieldValue>{$string}</fieldValue>
+    </field>
+    <field>
+      <fieldDefinitionIdentifier>image</fieldDefinitionIdentifier>
+      <languageCode>eng-GB</languageCode>
+      <fieldValue>
+            <value key="fileName">{$fileName}</value>
+            <value key="fileSize">{$fileSize}</value>
+            <value key="data">{$fileData}</value>
+      </fieldValue>
+    </field>
+  </fields>
+</ContentCreate>
+XML;
+        $request = $this->createHttpRequest(
+            'POST',
+            '/api/ezp/v2/content/objects',
+            'ContentCreate+xml',
+            'ContentInfo+json',
+            $body
+        );
+
+        $response = $this->sendHttpRequest($request);
+
+        self::assertEquals('201', $response->getStatusCode());
+        self::assertHttpResponseHasHeader($response, 'Location');
+        self::assertHttpResponseHasHeader($response, 'content-type', 'application/vnd.ez.api.ContentInfo+json');
+
+        $href = $response->getHeader('Location')[0];
+        $this->addCreatedElement($href);
+
+        return $href;
+    }
+
+    /**
+     * @depends testCreateContent
+     */
+    public function testPublishContent(string $restContentHref): string
+    {
+        $response = $this->sendHttpRequest(
+            $this->createHttpRequest('PUBLISH', sprintf('%s/versions/1', $restContentHref))
+        );
+        self::assertHttpResponseCodeEquals($response, 204);
+
+        return $restContentHref;
+    }
+
+    /**
+     * @depends testCreateContent
+     */
+    public function testLoadContent(string $restContentHref): void
+    {
+        $response = $this->sendHttpRequest(
+            $this->createHttpRequest(
+                'GET',
+                $restContentHref,
+                '',
+                'Version+json'
+            )
+        );
+
+        self::assertHttpResponseCodeEquals($response, 200);
+        self::assertArrayHasKey('content-type', $response->getHeaders());
+        self::assertHttpResponseHasHeader($response, 'content-type', 'application/vnd.ez.api.ContentInfo+json');
+    }
+
+    /**
+     * @depends testPublishContent
+     */
+    public function testGetImageVariation(string $hrefToImage): void
+    {
+        $restContentHref = $hrefToImage;
+        $imageResponse = $this->sendHttpRequest(
+            $this->createHttpRequest(
+                'GET',
+                $restContentHref . '/versions/1',
```

Comment:
following the previous changes:
```php
public function testGetImageVariation(string $restVersionHref): void
```
and then you can:
```suggestion
                $restVersionHref,
```